lib/config/index.js

```diff
@@ -0,0 +1,46 @@
+module.exports = getConfig
+
+var statSync = require('fs').statSync
+var path = require('path')
+
+var defaultsDeep = require('lodash').defaultsDeep
+var series = require('async').series
+
+var accountConfig = require('./account')
+var assureFolders = require('./assure-folders')
+var couchDbConfig = require('./db/couchdb')
+var getDefaults = require('./defaults')
+var pouchDbConfig = require('./db/pouchdb')
+var storeConfig = require('./store')
+
+function getConfig (server, config, callback) {
+  defaultsDeep(config, getDefaults())
+
+  var adapter = config.PouchDB.preferredAdapters[0]
```

Comment:
Providing the user does what this file did (https://github.com/hoodiehq/hoodie/blob/82897fd9da994e88f4a0aae30e7c6de02f04dad1/server/config/db/factory.js), then it works? (I'm about to test)